docs/pages/badge.js

```diff
@@ -25,51 +24,255 @@ export default function BadgePage({ generatedDocGen }: {| generatedDocGen: DocGe
+
+          5. **Neutral**
+          Indicates a general, non-critical status update. For example, unavailable or not started.
+ `}
+        >
+          <MainSection.Card
+            defaultCode={`
+  <Table accessibilityLabel="Type examples">
+    <Table.Header>
+      <Table.Row>
+        <Table.HeaderCell>
+          <Text weight="bold">Type</Text>
+        </Table.HeaderCell>
+        <Table.HeaderCell>
+          <Text weight="bold">Example</Text>
+        </Table.HeaderCell>
+      </Table.Row>
+    </Table.Header>
+    <Table.Body>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Info</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="New" type="info"/></Text>
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Success</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Complete" type="success"/></Text>
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Warning</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Needs attention" type="warning"/></Text>
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Error</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Failed" type="error"/></Text>
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Neutral</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Not started" type="neutral"/></Text>
+        </Table.Cell>
+      </Table.Row>
+    </Table.Body>
+  </Table>
 `}
-      />
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Over media"
+          description={`
+          Badge may be used over media using two wash styles.
 
-      <Example
-        description={`
-    Components like [Module](/module) and [Dropdown](/dropdown) support Badges within the component.`}
-        name="Example: within other components"
-        defaultCode={`
+          1. **Over media - Light wash**
+          The light wash badge should be used over media that is dark or utilizes a dark gradient overlay.
+
+          2. **Over media - Dark wash**
+          The dark wash badge should be used over media that is light or utilizes a light gradient overlay.
+`}
+        >
+          <MainSection.Card
+            defaultCode={`
+<Flex gap={4}>
+  <Mask height={150} width={280} rounding={2}>
+    <Image alt="Botanical art in coral and green" fit="cover" src="https://i.ibb.co/cbjgZft/img-door.jpg" naturalWidth={1} naturalHeight={1}/>
+    <Box position="absolute" top>
```

Comment:
This isn't necessary; elements to be overlaid on Image should be children of the component, and absolute positioning is built-in